.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -8,32 +8,109 @@ assignees: ''
 
 # Onboarding
 
-By creating this ticket, you are requesting to be onboarded as a developer on the Azul team.
-The action items in this ticket will be carried out by you and existing team
-members.
+This ticket should be created by a hiring supervisor to initiate the process of
+onboarding a employee/contractor to team Boardwalk. The action items in this
+ticket will be completed by you, existing team members, and the onboarded
+employee/contractor. Your next step is to assign this ticket to yourself.
 
-Your next steps are to carefully read the following documents:
+If you are assigned this ticket, perform the checklist items in order. Remove
+your assignment only after assigning this ticket to the next in line or the
+ticket checklist is completed. At that time, put the ticket back in triage.
 
-- [ ] [Azul README.md](..)
 
-- [ ] [Contributing Guidelines](../blob/develop/CONTRIBUTING.rst)
+## Hiring Supervisor 
 
-- [ ] [AWS Credential Configuration](https://giwiki.gi.ucsc.edu/index.php/Overview_of_Getting_and_Using_an_AWS_IAM_Account)
+- [ ] Onboarded employee/contractor's name: [ENTER-NAME-HERE]
+- [ ] Approved the HR request for ITS to create an UCSC account for the onboarded employee <sub>(or the person being onboarded is not an employee)</sub>
+- [ ] Instructed the onboarded employee/contractor to create a new GitHub account using the username `{ucsc_account_name}-ucsc` and connected to their UCSC account<sup>1</sup> email address. The new account will be used exclusively for work on the team and should be created even if they already have another account.
```

Comment:
Only applicable to UCSC employees. Contractors can continue to use their existing GH account.